app/models/user.rb

```diff
@@ -1,11 +1,13 @@
 class User < ApplicationRecord
   has_many :comments
   enum role: [:user, :admin]
-  
+
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable,
          :omniauthable, omniauth_providers: [:facebook]
   has_many :recipes
+  has_one :collection
+  after_create :make_collection
```

Comment:
What are the arguments that you create a collection this way? When the user is created I mean?

Reply:
By default every user has an empty collection, and therefor an instance of `collection` gets created for every new user.
Otherwise, we can create this `collection` instance for the user when he adds a recipe to his collection the first time.